checkov/sast/record.py

```diff
@@ -43,6 +45,7 @@ def __init__(self,
         self.cwe = cwe
         self.owasp = owasp
         self.show_severity = show_severity
+        self.dataflow = data_flow
```

Comment:
```suggestion
        self.data_flow = data_flow
```
no?

Reply:
I think its better to add a `metadata` that keeps the structure of our report, so its easier to extend or change in the future. So in this case its `metadata.taint_flow` (originaly its taint_mode but I think tait_flow is clearer)